include/ebpf_xdp_test_hooks.h

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) eBPF for Windows contributors
```

Comment:
Please move this to `tests\netebpfext_unit` or better yet consider adding this in `netebpfext_unit\netebpf_ext_helper.h`. Update all test code that needs xdp_md_test_t to include this header directly instead.

Reply:
xdp structure is used here:
netebpfext\net_ebpf_ext_program_info.h

Reply:
Understood. Then keep this inside the netebpfext directory for now as long as its not in the nuspec file.